src/test/java/StringTest.java

```diff
@@ -0,0 +1,55 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("split() 메서드 동작 Test")
+    public void splitTest() {
+        // given
+        String testString = "1,2";
+
+        // when
+        String[] splitTestArr = testString.split(",");
+
+        // then
+        assertThat(splitTestArr).contains("1");
+        assertThat(splitTestArr).containsExactly("1", "2");
+    }
+
+    @Test
+    @DisplayName("substringTest() 메서드 동작 Test")
+    void substringTest() {
+        // given
+        String testString = "(1,2)";
+
+        // when
+        String substringTestString = testString.substring(1, testString.length() - 1);
+
+        // then
+        assertThat(substringTestString).isEqualTo("1,2");
+    }
+
+    @DisplayName("charAt() 메서드 동작 Test_통과하는 {0,1,2} 와 실패하는 {3} 을 Exception 확인을 통해 함꼐 Test 진행")
+    @ParameterizedTest
+    @ValueSource(ints = {0,1,2,3})
+    void charAt(int index){
+        // given
+        String testString = "abc";
+        char[] checkStringArr = {'a', 'b', 'c'};
+
+        // when
+        try{
+            char alpha = testString.charAt(index);
+
+        // then
+            assertThat(alpha).isEqualTo(checkStringArr[index]);
+        } catch (StringIndexOutOfBoundsException e){
+            assertThat(e).hasMessage("String index out of range: 3");
+        }
```

Comment:
```suggestion
        assertThatThrownBy(() -> {
            testString.charAt(index);
        }).isInstanceOf(StringIndexOutOfBoundsException.class)
```
단순하게 이렇게 표현하면 된답니다!
단, 실패하는 케이스에 대해서만 테스트 하면 좋을 것 같아요~
지금은 정상, 비정상 input이 섞여있는 상태랍니다!